Header header

int NodeID
int CodeID
int Level
string Text
